platform-api/tests/test_form.py

```
@@ -0,0 +1,123 @@
+            "organization": "Acme",
+            "submittedAt": "2026-01-01T00:00:00Z",
+            "status": "submitted",
+            "formType": "contact"
+        }]
+    async def mutate(self, mutations: list[dict], dataset: str, write_token: str):
+        return {"results": [{"id": "new-doc"}]}
+
+@pytest.fixture
+def client(monkeypatch):
+    # Mock verify_turnstile so it doesn't make real HTTP requests during tests
+    async def fake_verify_turnstile(token, ip, secret):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `fake_verify_turnstile`

(ANN202)